src/client/messages.js

```diff
@@ -1,5 +1,8 @@
 export default {
   en: {
+    app: {
```

Comment:
I do like alphabetical order too. That's what is used for the ordering of translated components (2nd level). I just tried to follow the pattern of text order in the component on the following levels. Which is by my experience +- at the same useful level as alphabetical order (at least you don't have to resort texts after renaming placeholders). But non-alphabetical order can have one advantage => the order of placeholders can have some functional meaning. Like the msgArray, which get's you the messages in the exact order you want for your page. Of course this can be done via the naming of the keys, but I'm not so sure about this. Once you decide about msgArray function, it will be clearer, which order of keys to choose. :-)